shared/reducers/tabbed-router.js

```diff
@@ -29,28 +29,13 @@ const initialState: TabbedRouterState = Immutable.fromJS({
     [moreTab]: emptyRouterState,
     [loginTab]: emptyRouterState
   },
-  activeTab: LocalDebug.overrideActiveTab ? LocalDebug.overrideActiveTab : moreTab
-})
+  activeTab: moreTab
+}))
 
 export default function (state: TabbedRouterState = initialState, action: any): TabbedRouterState {
   switch (action.type) {
     case Constants.switchTab:
       return state.set('activeTab', action.payload)
-    case LoginConstants.loginDone:
```

Comment:
talked to @MarcoPolo  about this. the tabbed router shouldn't know about this kind of stuff